### ISSUE TITLE
Can't query MongoDB because of missing columns in some rows

### ISSUE THREAD

**Issue by smddzcy** [metabase/metabase#8929]
### Bugs
Currently, I can't query my MongoDB using Metabase. I get the following error for a super basic use case (I just select my table and try to view raw data):

```
Error: mismatched columns in results! Expected: [:_id :__v :x :y :z] Got: [:_id :x :y :z]
```

I believe it's related to Mongoose because the missing column `__v` is the version key that Mongoose puts automatically.

- Your browser and the version: Chrome 70.0
- Your operating system: OS X 10
- Your databases: MongoDB
- Metabase version: v0.31.0
- Metabase hosting environment: (e.x. Mac app, Elastic Beanstalk, Docker, Heroku, Linux/Ubuntu 12)
- Metabase internal database: (e.x. H2 (default), MySQL, Postgres)

Steps to reproduce:
- Click `Ask a question`
- Click `Custom`
- Select a MongoDB table that you use with Mongoose
- Click `Get Answer` (leave `Filtered by` and `Grouped by` empty, and `View` as `Raw data`)

**Comment by smddzcy** [comment]
Ugh, it's not specifically related to Mongoose, it's probably because there are some rows that don't have the `__v` column.

**Comment by camsaul** [comment]
Fixed by #8952
